lib/ws-kernel.js

```diff
@@ -27,21 +28,21 @@ export default class WSKernel extends Kernel {
   }
 
   interrupt() {
-    return this.session.kernel.interrupt();
+    this.session.kernel.interrupt();
   }
 
   shutdown() {
-    return this.session.kernel.shutdown();
+    this.session.kernel.shutdown();
   }
 
   restart(onRestarted: ?Function) {
     const future = this.session.kernel.restart();
     future.then(() => {
-      if (onRestarted) onRestarted(this.session.kernel);
+      if (onRestarted) onRestarted();
```

Comment:
why? This is an API change, it may brake functionality in Hydrogen.

Reply:
I ran `git grep` and double-checked every usage of restart before changing this.

Reply:
Nice finding!